### PR TITLE
Update 2022-02-16-this-week-in-rust.md (#1)

### PR DESCRIPTION
**Pull request by kw-fn**

### CODE REVIEW

**Comment by ericseppanen** [comment]
There are TWiR editors who already monitor most of the popular Rust sites (HN, Reddit, etc.) so there's no need for you to submit PRs for every link you come across. These add to the editor workload because we need to merge commits that contain things we're already adding.

If you come across a good link that's not already widely distributed, great! But most of these were already being tracked by the community editors.

**Comment by kw-fn** [comment]
Hi @ericseppanen. If you could provide documented guidance for which URLs are being monitored it would be helpful otherwise I don't see how I can distinguish what is contribution worthy content without potentially missing at least some relevant articles each week... 

Alternatively, if the prior PRs editing the weekly draft could be merged at a cadence faster than weekly at least the draft might then be used as a point in time reference to what has already been widely distributed.

Thanks!

**Comment by ericseppanen** [comment]
The things editors monitor is a moving target (and not every link from upstream sources is accepted), but it's safe to assume that if it's from a Rust core team member or gets reliably posted to Reddit or HN, we'll see it.

**Comment by nellshamrell** [comment]
Hello all -

The responsibility for the weekly cadence of merging pull requests (rather than reviewing and merging them as I come in) is on me. I'm realizing now that, with the volume that we receive, I've been making myself a bottleneck and it's been adding additional work to others. This is something that will be changing in the near future - my fellow editors will also be able to review and merge PRs. This should cut down on the review time and the merge conflicts.

**Comment by nellshamrell** [comment]
That said, @kw-fn, could you resolve the merge conflicts in this pull request? And it looks like this may be re-adding some links that were also added in just merged pull requests and those will need to be cleaned up.

File: draft/2022-02-16-this-week-in-rust.md [hunk 1]
```diff
@@ -22,13 +22,15 @@ If you find any errors in this week's issue, [please submit a PR](https://github
...
 
 ### Project/Tooling Updates
-* [This week in Fluvio #22: the programmable streaming platform](https://www.fluvio.io/news/this-week-in-fluvio-0022/)
-* [rsadsb: v0.5.0 Release](https://rsadsb.github.io/v0.5.0.html)
 * [Fornjot (Code-CAD in Rust) - Weekly Dev Log - 2022-W06](https://www.fornjot.app/blog/weekly-dev-log/2022-w06/)
+* [Performance Tools Newsletter (Q4 2021)](https://blog.mozilla.org/performance/2022/02/15/performance-tools-newsletter-q4-2021/)
+* [rsadsb: v0.5.0 Release](https://rsadsb.github.io/v0.5.0.html)
+* [SixtyFPS becomes Slint](https://slint-ui.com/blog/sixtyfps-becomes-slint.html)
+* [This week in Fluvio #22: the programmable streaming platform](https://www.fluvio.io/news/this-week-in-fluvio-0022/)
+* [This week in Glean](https://blog.mozilla.org/data/2022/02/09/this-week-in-glean-migrating-legacy-telemetry-collections-to-glean/)
```
Comment (nellshamrell) [inline draft/2022-02-16-this-week-in-rust.md:30]: While I recognize this project may have parts written in Rust, there's not enough Rust content in the newsletter to include this link.

**Review by nellshamrell — CHANGES_REQUESTED** [review]
After rebasing with master and removing the duplicates, it looks like the only two new articles are:

* [Performance Tools Newsletter (Q4 2021)](https://blog.mozilla.org/performance/2022/02/15/performance-tools-newsletter-q4-2021/)
*  [This week in Glean](https://blog.mozilla.org/data/2022/02/09/this-week-in-glean-migrating-legacy-telemetry-collections-to-glean/)

While the Performance Tools link is fine, the Glean one, as noted in the in line comments, does not have enough Rust content in it to include. Please remove the Glean one, and then I would be happy to merge this with the Performance tools link in it.

**Pull request merged**